src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java

```diff
@@ -325,7 +325,32 @@ public static boolean addToTooltip(List<String> tooltip, String internalname, It
 									if (shouldShow) {
 										tooltip.add(EnumChatFormatting.YELLOW.toString() + EnumChatFormatting.BOLD + "Required Items:");
 										for (JsonElement item : itemsObject.get(nextStarLevelString).getAsJsonArray()) {
-											tooltip.add("  - " + item.getAsString());
+											if (item.getAsString().contains("§")) {
+												//TODO show outdated repo notification when 2.1.1 releases
```

Comment:
Is this TODO for this PR or TODO in general for 2.1.1. otherwise lgtm

Reply:
TODO for 2.1.1, this is just backwards compat for the old format